wiki-thread-lighting-app.md

```diff
@@ -0,0 +1,108 @@
+and [form a Thread network](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#form-a-thread-network) 
+using the nRF52840 Dongle as the RCP. For details on setting up the RCP, 
+please refer to [Build and flash RCP firmware on nRF52480 dongle](https://github.com/canonical/openthread-border-router-snap/wiki/Setup-OpenThread-Border-Router-with-nRF52840-Dongle#build-and-flash-rcp-firmware-on-nrf52480-dongle).
+
+#### 2. Run a Matter Thread Lighting App on Machine B
+There are two options to run a Matter Thread lighting application. 
+
+**Option 1**: Native Built Lighting App
+
+You can choose this option to build and run the [Matter Thread lighting app example](https://github.com/project-chip/connectedhomeip/tree/6b01cb977127eb8547ce66d5b627061dc2dd6c90/examples/lighting-app/linux) natively. With this option, you can observe the lighting status (on/off) from the logs.
+
+To get started, follow these steps:
+
+- 1. Install the Matter SDK with a shallow clone for the Linux platform:
+```
+git clone https://github.com/project-chip/connectedhomeip.git --depth=1
+cd connectedhomeip
+git checkout 6b01cb977127eb8547ce66d5b627061dc2dd6c90
+scripts/checkout_submodules.py --shallow --platform linux
+```
+- 2. Build the lighting app:
+```
+cd examples/lighting-app/linux
+gn gen out/debug
+ninja -C out/debug
+```
+- 3. Run the lighting app with Thread feature enabled:
+```
+./out/debug/chip-lighting-app --thread
+```
+
+**Option 2**: GPIO Commander Snap
+
+Alternatively, you can use this option to set up the Matter Thread lighting application by installing the Matter Pi GPIO Commander snap. 
+This turns your Raspberry Pi into a Matter Thread lighting device, allowing you to control the connected LED, turning it on or off.
+
+Follow the instructions for installing the matter-pi-gpio-commander snap, 
+provided at [Setup OpenThread Border Router with nRF52840 Dongle](https://github.com/canonical/matter-pi-gpio-commander/wiki/Setup-and-control-a-lighting-device#installation).
+
+TBA: Then, running the GPIO commander snap with Thread feature enabled.
+
+#### 3. Run OpenThread Border Router Agent on Machine B
+Check the [install and configure the OTBR snap](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#install-and-configure-the-otbr-snap) section
+to run OTBR agent. The lighting app communicates with OTBR agent via the dbus message bus. This is necessary to support the lighting app's Thread feature, as explained
+[here](https://github.com/project-chip/connectedhomeip/tree/6b01cb977127eb8547ce66d5b627061dc2dd6c90/examples/lighting-app/linux#commandline-arguments).
+
+> Note: Only the OTBR agent is required, and the GUI (openthread-border-router.otbr-web) is optional. The OTBR agent can be started by running the following command:
+> ```bash
+> sudo snap start openthread-border-router.otbr-agent
+> ```
```

Comment:
The link on line 73 can be changed to a link to the README for installation and configuration. The rest of the needed operations can be documented here:
1. Install and config
2. Plugging the dongle
3. Starting the agent (for details, refer [here](https://github.com/canonical/openthread-border-router-snap#run))